backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/services/CoreFeaturesSwitch.java

```diff
@@ -140,6 +140,7 @@ public List<EStructuralFeature> caseOccurrenceUsage(OccurrenceUsage object) {
     public List<EStructuralFeature> caseMembership(Membership object) {
         var features = new ArrayList<EStructuralFeature>();
         features.add(SysmlPackage.eINSTANCE.getMembership_Visibility());
+        features.add(SysmlPackage.eINSTANCE.getMembership_MemberElement());
```

Comment:
Why add this property as Core property ?

Reply:
A Membership is used in the Expression for referencing the receiver element of the AcceptAction.
I thought it interesting to see this element.

Reply:
Ok but here you add the property on all Memberships, without regarding its parent or any other characteristic.
If you want, you can add this property on an AcceptAction as an extra property section in `SysMLv2PropertiesConfigurer`, it would be better.